lib/global-styles.php

```diff
@@ -314,6 +314,61 @@ function gutenberg_experimental_global_styles_get_block_data() {
 	return $block_data;
 }
 
+/**
+ * This function allows to easily access a part from a php array.
+ * It is equivalent to want lodash get provides for JavaScript and is useful to have something similar
+ * in php so functions that do the same thing on the client and sever can have identical code.
+ *
+ * @param array $array  An array from where we want to retrieve some information from.
+ * @param array $path   An array containing the path we want to retrieve.
+ *
+ * @return array Containing a set of css rules.
+ */
+function gutenberg_experimental_get( $array, $path ) {
```

Comment:
At the moment, we're using `gutenberg_experimental_global_styles_` as the prefix. This probably needs to change to split for the different functions, but that's best done when we cross that bridge.

Perhaps I'd name this as `gutenberg_experimental_global_styles_get_subtree` or `gutenberg_experimental_global_styles_get_deep_value`  or something like that, a bit more specific than `get`.

Reply:
The idea was for this eventually be a PHP version of get. There is nothing global styles specific in this function, it may be useful outside global styles. Maybe we should move the function to another file?

Reply:
Do we have a utility file for this kind of thing? That'd make sense.

At the same, given that it's only used here, it's perhaps more convenient to rename it to follow this file's conventions, at the moment?